.changeset/silent-flies-float.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": minor
+---
+
+Fixed: `declaration-block-no-shorthand-property-overrides` false negatives for `font` and `border`
```

Comment:
@ybiquitous should I

```suggestion
Fixed: `declaration-block-no-shorthand-property-overrides` false negatives for `font`
```

and add myself to https://github.com/stylelint/stylelint/pull/7582/files#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR8?

Reply:
Why removing `border`?

Reply:
I was thinking of replacing this line

```md
- Fixed: `declaration-block-no-shorthand-property-overrides` false negatives for `border` ([#7585](https://github.com/stylelint/stylelint/pull/7585)) ([@fpetrakov](https://github.com/fpetrakov)).
```

by

```md
- Fixed: `declaration-block-no-shorthand-property-overrides` false negatives for `border` ([#7585](https://github.com/stylelint/stylelint/pull/7585) & [#7606](https://github.com/stylelint/stylelint/pull/7606)) ([@fpetrakov](https://github.com/fpetrakov) & [@Mouvedia](https://github.com/Mouvedia)).
```